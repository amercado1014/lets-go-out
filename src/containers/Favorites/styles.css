.favorites {
  height: 100vh;
  color: white;
}

.favorites-message {
  color: white;
  background-color: #20161C;
  font-size: 20px;
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 10% auto;
  width: 50vw;
  height: 40%;
}