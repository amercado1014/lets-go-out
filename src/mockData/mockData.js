export const mockRestaurantsApiData = {
  "address": {
    "apiKey": null,
    "streetAddress": "",
    "latitude": 39.6682515,
    "longitude": -105.096661,
    "city": "Denver",
    "state": "CO",
    "zip": "80227",
    "aptNumber": null
  },
  "restaurants": [
    {
      "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f31932e981e6050b6c",
      "deliveryMin": 0,
      "deliveryPrice": 3.5,
      "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-s-wadsworth-21136_1407870747356.png",
      "name": "Big Daddy's Pizza - S. Wadsworth",
      "streetAddress": "2772 S Wadsworth Blvd",
      "city": "Denver",
      "state": "CO",
      "zip": "80227",
      "foodTypes": [
        "Pizza"
      ],
      "phone": "(303) 988-3697",
      "latitude": 39.666141,
      "longitude": -105.080677,
      "minFreeDelivery": 0,
      "taxRate": 0.08,
      "acceptsCash": true,
      "acceptsCard": true,
      "offersPickup": true,
      "offersDelivery": true,
      "isTestRestaurant": false,
      "minWaitTime": 45,
      "maxWaitTime": 60,
      "open": false,
      "url": "https://eatstreet.com/denver-co/restaurants/big-daddys-pizza-s-wadsworth",
      "hours": {
        "Monday": [
          "11:00 AM - 3:00 AM"
        ],
        "Saturday": [
          "11:00 AM - 3:00 AM"
        ],
        "Sunday": [
          "11:00 AM - 3:00 AM"
        ],
        "Wednesday": [
          "11:00 AM - 3:00 AM"
        ],
        "Tuesday": [
          "11:00 AM - 3:00 AM"
        ],
        "Friday": [
          "11:00 AM - 3:00 AM"
        ],
        "Thursday": [
          "11:00 AM - 3:00 AM"
        ]
      },
      "timezone": "US/Mountain"
    }
  ]
};

export const returnedMockRestaurantApiData = [
  {
    "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f31932e981e6050b6c",
    "deliveryMin": 0,
    "deliveryPrice": 3.5,
    "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-s-wadsworth-21136_1407870747356.png",
    "name": "Big Daddy's Pizza - S. Wadsworth",
    "streetAddress": "2772 S Wadsworth Blvd",
    "city": "Denver",
    "state": "CO",
    "zip": "80227",
    "foodTypes": [
      "Pizza"
    ],
    "phone": "(303) 988-3697",
    "latitude": 39.666141,
    "longitude": -105.080677,
    "minFreeDelivery": 0,
    "taxRate": 0.08,
    "acceptsCash": true,
    "acceptsCard": true,
    "offersPickup": true,
    "offersDelivery": true,
    "isTestRestaurant": false,
    "minWaitTime": 45,
    "maxWaitTime": 60,
    "open": false,
    "url": "https://eatstreet.com/denver-co/restaurants/big-daddys-pizza-s-wadsworth",
    "hours": {
      "Monday": [
        "11:00 AM - 3:00 AM"
      ],
      "Saturday": [
        "11:00 AM - 3:00 AM"
      ],
      "Sunday": [
        "11:00 AM - 3:00 AM"
      ],
      "Wednesday": [
        "11:00 AM - 3:00 AM"
      ],
      "Tuesday": [
        "11:00 AM - 3:00 AM"
      ],
      "Friday": [
        "11:00 AM - 3:00 AM"
      ],
      "Thursday": [
        "11:00 AM - 3:00 AM"
      ]
    },
    "timezone": "US/Mountain"
  }
];

export const cleanRestaurantData = {
  apiKey: "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
  logoUrl: "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
  name: "Big Daddy's Pizza - Lakewood",
  streetAddress: "850 Wadsworth Blvd",
  city: "Lakewood",
  state: "CO",
  zip: "80214",
  foodTypes: "Pizza",
  phone: "(303) 736-6090",
  url: "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
  hours: {
    Monday: "11:00 AM - 3:00 AM",
    Saturday: "11:00 AM - 3:00 AM",
    Sunday: "11:00 AM - 3:00 AM",
    Wednesday: "11:00 AM - 3:00 AM",
    Tuesday: "11:00 AM - 3:00 AM",
    Friday: "11:00 AM - 3:00 AM",
    Thursday: "11:00 AM - 3:00 AM"
  }
};