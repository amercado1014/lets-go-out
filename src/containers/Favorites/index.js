import React from 'react';
import './styles.css';
import Restaurants from '../Restaurants';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';

export const Favorites = ({favorites}) => {
  const displayFavorites = favorites.map(favorite => {
    return <Restaurants key={favorite.apiKey} 
      restaurant={favorite}
      isFavorite={true} />;
  });

  return (
    <div>
      <div className="favorites">
        {displayFavorites.length > 0 
          ? displayFavorites 
          : <h4>There are no favorites to display.</h4>}
      </div>
    </div>
  );
};

export const mapStateToProps = state => ({
  favorites: state.favorites
});

Favorites.propTypes = {
  favorites: PropTypes.array
};

export default connect(mapStateToProps)(Favorites);