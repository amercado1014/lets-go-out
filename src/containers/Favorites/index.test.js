import React from 'react';
import { Favorites, mapStateToProps } from '../Favorites';
import { shallow } from 'enzyme';
import { mockRestaurants } from '../../mockData/mockData';

describe('Favorites', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Favorites favorites={mockRestaurants} />);
  });

  it('should math the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

describe('mapStateToProps', () => {
  it('correctly maps restaurants to props', () => {
    const favorites = mockRestaurants;
    const expected = favorites;
    const mockState = { favorites };
    const mapped = mapStateToProps(mockState);
    expect(mapped.favorites).toEqual(expected);
  });
});
