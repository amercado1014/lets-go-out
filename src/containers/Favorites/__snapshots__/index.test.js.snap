// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites should math the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    favorites={
        Array [
            Object {
              "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
              "city": "Lakewood",
              "deliveryMin": 12,
              "foodTypes": "Pizza",
              "hours": Object {
                "Friday": "11:00 AM - 3:00 AM",
                "Monday": "11:00 AM - 3:00 AM",
                "Saturday": "11:00 AM - 3:00 AM",
                "Sunday": "11:00 AM - 3:00 AM",
                "Thursday": "11:00 AM - 3:00 AM",
                "Tuesday": "11:00 AM - 3:00 AM",
                "Wednesday": "11:00 AM - 3:00 AM",
              },
              "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
              "minFreeDelivery": 0,
              "name": "Big Daddy's Pizza - Lakewood",
              "offersDelivery": false,
              "phone": "(303) 736-6090",
              "state": "CO",
              "streetAddress": "850 Wadsworth Blvd",
              "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
              "zip": "80214",
            },
            Object {
              "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
              "city": "Lakewood",
              "deliveryMin": 12,
              "foodTypes": "Pizza",
              "hours": Object {
                "Friday": "11:00 AM - 3:00 AM",
                "Monday": "11:00 AM - 3:00 AM",
                "Saturday": "11:00 AM - 3:00 AM",
                "Sunday": "11:00 AM - 3:00 AM",
                "Thursday": "11:00 AM - 3:00 AM",
                "Tuesday": "11:00 AM - 3:00 AM",
                "Wednesday": "11:00 AM - 3:00 AM",
              },
              "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
              "minFreeDelivery": 0,
              "name": "Big Daddy's Pizza - Lakewood",
              "offersDelivery": false,
              "phone": "(303) 736-6090",
              "state": "CO",
              "streetAddress": "850 Wadsworth Blvd",
              "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
              "zip": "80214",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="favorites"
>
        <div
                className="close-div"
        >
                <Link
                        className="close-btn"
                        replace={false}
                        to="/"
                >
                        ◀ Back to Restaurants
                </Link>
        </div>
        <withRouter(Connect(Restaurants))
                isFavorite={true}
                restaurant={
                        Object {
                                "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                "city": "Lakewood",
                                "deliveryMin": 12,
                                "foodTypes": "Pizza",
                                "hours": Object {
                                  "Friday": "11:00 AM - 3:00 AM",
                                  "Monday": "11:00 AM - 3:00 AM",
                                  "Saturday": "11:00 AM - 3:00 AM",
                                  "Sunday": "11:00 AM - 3:00 AM",
                                  "Thursday": "11:00 AM - 3:00 AM",
                                  "Tuesday": "11:00 AM - 3:00 AM",
                                  "Wednesday": "11:00 AM - 3:00 AM",
                                },
                                "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                "minFreeDelivery": 0,
                                "name": "Big Daddy's Pizza - Lakewood",
                                "offersDelivery": false,
                                "phone": "(303) 736-6090",
                                "state": "CO",
                                "streetAddress": "850 Wadsworth Blvd",
                                "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                "zip": "80214",
                              }
                }
        />
        <withRouter(Connect(Restaurants))
                isFavorite={true}
                restaurant={
                        Object {
                                "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                "city": "Lakewood",
                                "deliveryMin": 12,
                                "foodTypes": "Pizza",
                                "hours": Object {
                                  "Friday": "11:00 AM - 3:00 AM",
                                  "Monday": "11:00 AM - 3:00 AM",
                                  "Saturday": "11:00 AM - 3:00 AM",
                                  "Sunday": "11:00 AM - 3:00 AM",
                                  "Thursday": "11:00 AM - 3:00 AM",
                                  "Tuesday": "11:00 AM - 3:00 AM",
                                  "Wednesday": "11:00 AM - 3:00 AM",
                                },
                                "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                "minFreeDelivery": 0,
                                "name": "Big Daddy's Pizza - Lakewood",
                                "offersDelivery": false,
                                "phone": "(303) 736-6090",
                                "state": "CO",
                                "streetAddress": "850 Wadsworth Blvd",
                                "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                "zip": "80214",
                              }
                }
        />
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="close-div"
>
            <Link
                        className="close-btn"
                        replace={false}
                        to="/"
            >
                        ◀ Back to Restaurants
            </Link>
</div>,
          Array [
            <withRouter(Connect(Restaurants))
              isFavorite={true}
              restaurant={
                            Object {
                                          "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                          "city": "Lakewood",
                                          "deliveryMin": 12,
                                          "foodTypes": "Pizza",
                                          "hours": Object {
                                            "Friday": "11:00 AM - 3:00 AM",
                                            "Monday": "11:00 AM - 3:00 AM",
                                            "Saturday": "11:00 AM - 3:00 AM",
                                            "Sunday": "11:00 AM - 3:00 AM",
                                            "Thursday": "11:00 AM - 3:00 AM",
                                            "Tuesday": "11:00 AM - 3:00 AM",
                                            "Wednesday": "11:00 AM - 3:00 AM",
                                          },
                                          "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                          "minFreeDelivery": 0,
                                          "name": "Big Daddy's Pizza - Lakewood",
                                          "offersDelivery": false,
                                          "phone": "(303) 736-6090",
                                          "state": "CO",
                                          "streetAddress": "850 Wadsworth Blvd",
                                          "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                          "zip": "80214",
                                        }
              }
/>,
            <withRouter(Connect(Restaurants))
              isFavorite={true}
              restaurant={
                            Object {
                                          "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                          "city": "Lakewood",
                                          "deliveryMin": 12,
                                          "foodTypes": "Pizza",
                                          "hours": Object {
                                            "Friday": "11:00 AM - 3:00 AM",
                                            "Monday": "11:00 AM - 3:00 AM",
                                            "Saturday": "11:00 AM - 3:00 AM",
                                            "Sunday": "11:00 AM - 3:00 AM",
                                            "Thursday": "11:00 AM - 3:00 AM",
                                            "Tuesday": "11:00 AM - 3:00 AM",
                                            "Wednesday": "11:00 AM - 3:00 AM",
                                          },
                                          "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                          "minFreeDelivery": 0,
                                          "name": "Big Daddy's Pizza - Lakewood",
                                          "offersDelivery": false,
                                          "phone": "(303) 736-6090",
                                          "state": "CO",
                                          "streetAddress": "850 Wadsworth Blvd",
                                          "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                          "zip": "80214",
                                        }
              }
/>,
          ],
        ],
        "className": "favorites",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="close-btn"
              replace={false}
              to="/"
>
              ◀ Back to Restaurants
</Link>,
            "className": "close-div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "◀ Back to Restaurants",
              "className": "close-btn",
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": "◀ Back to Restaurants",
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
          "nodeType": "function",
          "props": Object {
            "isFavorite": true,
            "restaurant": Object {
              "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
              "city": "Lakewood",
              "deliveryMin": 12,
              "foodTypes": "Pizza",
              "hours": Object {
                "Friday": "11:00 AM - 3:00 AM",
                "Monday": "11:00 AM - 3:00 AM",
                "Saturday": "11:00 AM - 3:00 AM",
                "Sunday": "11:00 AM - 3:00 AM",
                "Thursday": "11:00 AM - 3:00 AM",
                "Tuesday": "11:00 AM - 3:00 AM",
                "Wednesday": "11:00 AM - 3:00 AM",
              },
              "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
              "minFreeDelivery": 0,
              "name": "Big Daddy's Pizza - Lakewood",
              "offersDelivery": false,
              "phone": "(303) 736-6090",
              "state": "CO",
              "streetAddress": "850 Wadsworth Blvd",
              "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
              "zip": "80214",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
          "nodeType": "function",
          "props": Object {
            "isFavorite": true,
            "restaurant": Object {
              "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
              "city": "Lakewood",
              "deliveryMin": 12,
              "foodTypes": "Pizza",
              "hours": Object {
                "Friday": "11:00 AM - 3:00 AM",
                "Monday": "11:00 AM - 3:00 AM",
                "Saturday": "11:00 AM - 3:00 AM",
                "Sunday": "11:00 AM - 3:00 AM",
                "Thursday": "11:00 AM - 3:00 AM",
                "Tuesday": "11:00 AM - 3:00 AM",
                "Wednesday": "11:00 AM - 3:00 AM",
              },
              "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
              "minFreeDelivery": 0,
              "name": "Big Daddy's Pizza - Lakewood",
              "offersDelivery": false,
              "phone": "(303) 736-6090",
              "state": "CO",
              "streetAddress": "850 Wadsworth Blvd",
              "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
              "zip": "80214",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="favorites"
>
          <div
                    className="close-div"
          >
                    <Link
                              className="close-btn"
                              replace={false}
                              to="/"
                    >
                              ◀ Back to Restaurants
                    </Link>
          </div>
          <withRouter(Connect(Restaurants))
                    isFavorite={true}
                    restaurant={
                              Object {
                                        "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                        "city": "Lakewood",
                                        "deliveryMin": 12,
                                        "foodTypes": "Pizza",
                                        "hours": Object {
                                          "Friday": "11:00 AM - 3:00 AM",
                                          "Monday": "11:00 AM - 3:00 AM",
                                          "Saturday": "11:00 AM - 3:00 AM",
                                          "Sunday": "11:00 AM - 3:00 AM",
                                          "Thursday": "11:00 AM - 3:00 AM",
                                          "Tuesday": "11:00 AM - 3:00 AM",
                                          "Wednesday": "11:00 AM - 3:00 AM",
                                        },
                                        "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                        "minFreeDelivery": 0,
                                        "name": "Big Daddy's Pizza - Lakewood",
                                        "offersDelivery": false,
                                        "phone": "(303) 736-6090",
                                        "state": "CO",
                                        "streetAddress": "850 Wadsworth Blvd",
                                        "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                        "zip": "80214",
                                      }
                    }
          />
          <withRouter(Connect(Restaurants))
                    isFavorite={true}
                    restaurant={
                              Object {
                                        "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                        "city": "Lakewood",
                                        "deliveryMin": 12,
                                        "foodTypes": "Pizza",
                                        "hours": Object {
                                          "Friday": "11:00 AM - 3:00 AM",
                                          "Monday": "11:00 AM - 3:00 AM",
                                          "Saturday": "11:00 AM - 3:00 AM",
                                          "Sunday": "11:00 AM - 3:00 AM",
                                          "Thursday": "11:00 AM - 3:00 AM",
                                          "Tuesday": "11:00 AM - 3:00 AM",
                                          "Wednesday": "11:00 AM - 3:00 AM",
                                        },
                                        "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                        "minFreeDelivery": 0,
                                        "name": "Big Daddy's Pizza - Lakewood",
                                        "offersDelivery": false,
                                        "phone": "(303) 736-6090",
                                        "state": "CO",
                                        "streetAddress": "850 Wadsworth Blvd",
                                        "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                        "zip": "80214",
                                      }
                    }
          />
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="close-div"
>
              <Link
                            className="close-btn"
                            replace={false}
                            to="/"
              >
                            ◀ Back to Restaurants
              </Link>
</div>,
            Array [
              <withRouter(Connect(Restaurants))
                isFavorite={true}
                restaurant={
                                Object {
                                                "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                                "city": "Lakewood",
                                                "deliveryMin": 12,
                                                "foodTypes": "Pizza",
                                                "hours": Object {
                                                  "Friday": "11:00 AM - 3:00 AM",
                                                  "Monday": "11:00 AM - 3:00 AM",
                                                  "Saturday": "11:00 AM - 3:00 AM",
                                                  "Sunday": "11:00 AM - 3:00 AM",
                                                  "Thursday": "11:00 AM - 3:00 AM",
                                                  "Tuesday": "11:00 AM - 3:00 AM",
                                                  "Wednesday": "11:00 AM - 3:00 AM",
                                                },
                                                "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                                "minFreeDelivery": 0,
                                                "name": "Big Daddy's Pizza - Lakewood",
                                                "offersDelivery": false,
                                                "phone": "(303) 736-6090",
                                                "state": "CO",
                                                "streetAddress": "850 Wadsworth Blvd",
                                                "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                                "zip": "80214",
                                              }
                }
/>,
              <withRouter(Connect(Restaurants))
                isFavorite={true}
                restaurant={
                                Object {
                                                "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                                                "city": "Lakewood",
                                                "deliveryMin": 12,
                                                "foodTypes": "Pizza",
                                                "hours": Object {
                                                  "Friday": "11:00 AM - 3:00 AM",
                                                  "Monday": "11:00 AM - 3:00 AM",
                                                  "Saturday": "11:00 AM - 3:00 AM",
                                                  "Sunday": "11:00 AM - 3:00 AM",
                                                  "Thursday": "11:00 AM - 3:00 AM",
                                                  "Tuesday": "11:00 AM - 3:00 AM",
                                                  "Wednesday": "11:00 AM - 3:00 AM",
                                                },
                                                "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                                                "minFreeDelivery": 0,
                                                "name": "Big Daddy's Pizza - Lakewood",
                                                "offersDelivery": false,
                                                "phone": "(303) 736-6090",
                                                "state": "CO",
                                                "streetAddress": "850 Wadsworth Blvd",
                                                "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                                                "zip": "80214",
                                              }
                }
/>,
            ],
          ],
          "className": "favorites",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="close-btn"
                replace={false}
                to="/"
>
                ◀ Back to Restaurants
</Link>,
              "className": "close-div",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "◀ Back to Restaurants",
                "className": "close-btn",
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "◀ Back to Restaurants",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
            "nodeType": "function",
            "props": Object {
              "isFavorite": true,
              "restaurant": Object {
                "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                "city": "Lakewood",
                "deliveryMin": 12,
                "foodTypes": "Pizza",
                "hours": Object {
                  "Friday": "11:00 AM - 3:00 AM",
                  "Monday": "11:00 AM - 3:00 AM",
                  "Saturday": "11:00 AM - 3:00 AM",
                  "Sunday": "11:00 AM - 3:00 AM",
                  "Thursday": "11:00 AM - 3:00 AM",
                  "Tuesday": "11:00 AM - 3:00 AM",
                  "Wednesday": "11:00 AM - 3:00 AM",
                },
                "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                "minFreeDelivery": 0,
                "name": "Big Daddy's Pizza - Lakewood",
                "offersDelivery": false,
                "phone": "(303) 736-6090",
                "state": "CO",
                "streetAddress": "850 Wadsworth Blvd",
                "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                "zip": "80214",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
            "nodeType": "function",
            "props": Object {
              "isFavorite": true,
              "restaurant": Object {
                "apiKey": "aa1163f8eb68a8f7144247b91fbcb2f39e65e39821ec08a9",
                "city": "Lakewood",
                "deliveryMin": 12,
                "foodTypes": "Pizza",
                "hours": Object {
                  "Friday": "11:00 AM - 3:00 AM",
                  "Monday": "11:00 AM - 3:00 AM",
                  "Saturday": "11:00 AM - 3:00 AM",
                  "Sunday": "11:00 AM - 3:00 AM",
                  "Thursday": "11:00 AM - 3:00 AM",
                  "Tuesday": "11:00 AM - 3:00 AM",
                  "Wednesday": "11:00 AM - 3:00 AM",
                },
                "logoUrl": "https://eatstreet-static.s3.amazonaws.com/assets/images/restaurant_logos/big-daddys-pizza-lakewood-21135_1407868711096.png",
                "minFreeDelivery": 0,
                "name": "Big Daddy's Pizza - Lakewood",
                "offersDelivery": false,
                "phone": "(303) 736-6090",
                "state": "CO",
                "streetAddress": "850 Wadsworth Blvd",
                "url": "https://eatstreet.com/lakewood-co/restaurants/big-daddys-pizza-lakewood",
                "zip": "80214",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
